src/screens/CreatePost/index.js

```diff
@@ -449,11 +450,13 @@ const CreatePost = () => {
   };
 
   const postV2 = async () => {
+    setLoadingPost(true);
     if (!isEmptyMessageAllowed()) {
       showMessage({
         message: StringConstant.createPostFailedNoMessage,
         type: 'danger'
       });
+      setLoadingPost(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This block of code sets the `loadingPost` state to true at the start of the `postV2` function and then immediately checks if `isEmptyMessageAllowed()` returns false. If it does, it shows a message and sets `loadingPost` back to false. This is a good practice as it ensures the loading state is correctly managed even when the post creation cannot proceed due to empty messages.
```diff
+    setLoadingPost(true);
+    if (!isEmptyMessageAllowed()) {
+      showMessage({
+        message: StringConstant.createPostFailedNoMessage,
+        type: 'danger'
+      });
+      setLoadingPost(false);
+      return true;
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->